lib/test/drm/cenc/getLicenseServerUrlFromContentProtection.test.ts

```diff
@@ -43,13 +40,13 @@ describe('getLicenseServerUrlFromContentProtection', () => {
 	});
 
 	it('should return license server URL for clearkey prefix', () => {
-		contentProtection[0].laUrl.__prefix = 'clearkey';
+		// contentProtection[0].laUrl.__prefix = 'clearkey';
```

Comment:
@agajassi What is the purpose of the `__prefix` code? I don't see the `getLicenseServerUrlFromContentProtection` using that property in its logic. Can we remove these lines?